### ISSUE TITLE
clean install but infinite loop

### ISSUE THREAD

**Issue by sjtuross** [Nico640/docker-unms#79]
I tried the latest 1.4.8 in both arm and x86 environment but encountered an infinite loop issue. Before every retry, I got everything deleted in the config folder. It always enters a new loop after the following log.

```
2022-12-12T11:18:19.806975711+08:00 {"message":"UNMS did not start in time, restarting container.","channel":"web.sh","datetime":"2022-12-12T11:18:19+08:00","severity":"WARNING","level":300}
2022-12-12T11:18:19.807637215+08:00 make: *** [Makefile:22: server_with_migrate] Error 1
2022-12-12T11:18:19.813892857+08:00 2022-12-12 11:18:19.813 CST [1939] FATAL:  role "root" does not exist
```

Below is the docker-compose.yml and the log. I can't figure out what could be the cause. Could you please take a look? Thanks a lot.

```
version: '2'

services:
  uisp:
    image: nico640/docker-unms:latest
    container_name: uisp-nico640
    restart: unless-stopped
    ports:
      - 8443:8443
    environment:
      - PUBLIC_HTTPS_PORT=8443
      - PUBLIC_WS_PORT=8443
      - PUID=1000
      - PGID=1000
      - TZ=Asia/Shanghai
    volumes:
      - ./config:/config
```

[uisp-nico640-2022-12-12T03-18-57.log](https://github.com/Nico640/docker-unms/files/10204336/uisp-nico640-2022-12-12T03-18-57.log)

**Comment by Nico640** [comment]
Hi, that log is actually nothing out ouf the ordinary. UNMS is not started completely because it is waiting for the installation wizard to be completed. Therefore, UCRM will restart if UNMS doesn't start in that time span. But that shouldn't be a problem. As far as I can see, UNMS started correctly and is waiting to be setup.

I think your problem is that the port is not correct. The port inside the container is **443**, however, you are mapping both the external port (your host) and the internal port (the container) to port 8443. The correct way should be `8443:443`, everything else seems to be fine.

So basically, the docker-compose.yml should look like this

```
version: '2'

services:
  uisp:
    image: nico640/docker-unms:latest
    container_name: uisp-nico640
    restart: unless-stopped
    ports:
      - 8443:443
    environment:
      - PUBLIC_HTTPS_PORT=8443
      - PUBLIC_WS_PORT=8443
      - PUID=1000
      - PGID=1000
      - TZ=Asia/Shanghai
    volumes:
      - ./config:/config
```

**Comment by sjtuross** [comment]
Thank you so much for the detailed explanation and the work itself! I misunderstood PUBLIC_HTTPS_PORT and was not sure if some errors from log is normal or not. It's working perfectly now. Thanks again.
